### ISSUE TITLE
ESlint expo config breaking third party libraries

### ISSUE THREAD

**Issue by geminiyellow** [VISI-ONE/create-react-native-web-app#18]
**Describe the bug**
1. use v2 create a new project
2. copy some code from react-native-elements example
3. boom

```
> node scripts/build.js

Creating an optimized production build...
Failed to compile.

./node_modules/react-native-elements/src/helpers/getIconType.js
Cannot find module: '@expo/vector-icons/Feather'. Make sure this package is installed.

You can install this package by running: yarn add @expo/vector-icons/Feather.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! creaternwapp@0.1.0 build: `node scripts/build.js`
npm ERR! Exit status 1
```

**Expected behavior**

1. i checked the `react-native-elements`'s code, and there is no `@expo` dependence.
2. is it cause by `'babel-preset-expo'` ?

**Screenshots**
![image](https://user-images.githubusercontent.com/929878/49491296-fdb3c500-f896-11e8-9cb5-91e6b1e5f9b0.png)

**Comment by orYoffe** [comment]
I'm sorry but i don't know this library `react-native-elements` maybe it's better to ask them about this problem. also this project is for native and web and I'm not sure this library works for react -native-web as well

I hope you find your answer :)

**Comment by geminiyellow** [comment]
hi @orYoffe , `rne` is working at v1. 
i think the problem is the 

> is it cause by 'babel-preset-expo' ?

i delete all the plugin change it like this:

```
module.exports = function (api) {
  api.cache(true);
  return {
    "presets": [
      'module:metro-react-native-babel-preset'
    ],
  };
};

```

it work again.

this cli not for `expo`,  could you tell me why use the `babel-preset-expo` ?

**Comment by orYoffe** [comment]
hi @geminiyellow,
there is no special reason, I basically merged `create-react-app` and `create-react-native-app`. `babel-preset-expo` came from `create-react-native-app` so i don't mind if it works with web, ios and android. Also the user can change it themselves. 
Actually i would like to remove expo completely from the repo :)

would you like to create a pull request for this and contribute to the project or would you prefer to wait until i will do it? :)

**Comment by max-l** [comment]
I was wondering why the babel-preset-expo preset was in the config, if there is no reason I would remove it.  react-native-web have already require many parts working together, if there can be one less ;-)

**Comment by orYoffe** [comment]
@max-l The it is in the config from cerate-react-native-app CLI.. I will release a fix soon thanks

**Comment by orYoffe** [comment]
fixed with v0.1.10
